transports/websocket/Cargo.toml

```diff
@@ -16,7 +16,7 @@ either = "1.5.3"
 futures = "0.3.1"
 libp2p-core = { version = "0.16.0", path = "../../core" }
 log = "0.4.8"
-quicksink = "0.1"
+quicksink = { git = "https://github.com/paritytech/quicksink", branch = "develop" }
```

Comment:
If this PR is approved, the new quicksink version will be published to crates.io and this entry will become `quicksink = "0.2"`.